x-pack/dockerlogbeat/readme.md

```diff
@@ -1,10 +1,10 @@
-# dockerlogbeat: a simple, a docker plugin for sending logs to elasticsearch
+# Elastic Logging Plugin: a docker plugin for sending logs to elasticsearch
 
 
-This code is a working alpha for a [docker logging plugin](https://docs.docker.com/engine/extend/plugins_logging/). With the proper config, it can send logs to elasticsearch.
+This code is a working MVP for a [docker logging plugin](https://docs.docker.com/engine/extend/plugins_logging/). With the proper config, it can send logs to elasticsearch.
 ## Build and install
 
-To build and install, just run `mage create`
+To build and install, just run `mage Package`
```

Comment:
mage package?

Reply:
Yah, I changed the name to be more consistent with the rest of the beats.

Reply:
I think what Kaiyan meant was you to use lowercase here, as this is how all beats expose this in mage help (btw both upper and lower cases seem to work). For example:

```
X1 :: elastic/beats/metricbea » mage                                                                                                                                                                              2019/11/25 15:53:19 Found Elastic Beats dir at /home/exekias/go/src/github.com/elastic/beats
Targets:
  ...
  package                           packages the Beat for distribution.
  ...
```